writing/milestone-0/OVERVIEW.md

```diff
@@ -0,0 +1,80 @@
+## Overview
+Substrate is a new kind of computer console specialized for running AI at home and work
+
+The console is paired with an open source operating system and suite of applications. Our goal is to make both hardware and software radically accessible —  to teach the world not just how to use the AI and appliance, but how they can make (and re-make!) it themselves.
+
+Our first iteration of the console will be a kit that people can build in their own home or office.
+
+Today, upgrading a computer usually means replacing components — but our unique approach is to design modular systems users can upgrade through expansion. 
+
+In practice this means a single initial machine with cables that can be plugged into one or more "expansion boxes". People can invest in expanding these systems over many years. With each expansion, the system can assist in more activities and bring more smarts to each of them.
+
+We have plans for single-, double-, and quad-accelerator expansion boxes that can be plugged into a variety of machines. The plans will be available in early 2024. The double reuses all the parts of the single, and the quad reuses all the parts of the single and double.
+Who It's For
+Initially, this project will serve hackers who want to build and collaborate with one another on local and intelligent systems. 
+
+### Who is it for? What are some sample situations?
+
+A1. Hackers (working on an open source software/hardware project)
+
+A2. Students (in grad school doing research, with international and multilingual collaborators)
+
+A3. Small and mid-sized businesses (a regional distributor)
+
+
+### What It Is
+
+#### What you get in the mail:
+- Substrate System 0 is an easy-to-assemble AI computer kit for home and work.
+
+#### Once built, it runs:
+- SubstrateOS, an operating system that boots from a USB drive and schedules AI models on internal and external consumer GPUs.
+
+#### Once booted, it can:
+- M0.1. Join in on real-world conversations via your other devices. [A1] [A2] [A3]
+    A1. Planning meeting
+    A2. Lecture
+    A3. Sales meeting, customer call
+
+- M0.2. Take meeting and lecture notes so you can pay full attention to the speakers. [Liv] [A1] [A2] [A3]
+    A1. Meeting minutes, next steps
+    A2. Lecture notes
+    A3. ?
+
+- M0.3. Identify and link to more info on people, events, and cited documents. [Liv] [A1] [A2]
+    A1. ?
+    A2. Important dates for the course; references to other talks, textbooks, papers, textbooks
+
+- M0.4. Keep an at-hand, ever-present record of conversation, tasks, and specific contributions. This captures good ideas as they happen, foster new connections between them, and helps maintain understanding between collaborators. [Matt] [Liv] [A1] [A2] [A3]
+    A1. Feature suggestions, observations, 
+    A2. ?
+    A3. ?
+
+- M0.5. Eliminate language barriers by translating between any number of supported languages in real-time, across speech and text. [A2] [A3]
+    A2. ?
```

Comment:
A2: Enable access to a greater library of educational content and collaborators from institutions that do not share the student's native language

_Example Context: Columbia Business School has a requirement for graduate students to do international seminars. The courses are taught by CBS staff in English, but take place in countries where the native language is not English. By being able to access M0.5 capabilities with Bridge, students would have a more comprehensive exposure to the business ecosystem for their seminars._

Reply:
This is a great example. Can you be more specific about a situation that you imagine you might use bridge in to satisfy this educational requirement?